docs/src/charmil_factory.md

```diff
@@ -0,0 +1,36 @@
+## Charmil Factory
+The Charmil Factory offers you one-stop access to all of the Charmil packages all at once. So you just need to initialize factory in your cobra command and start using logging, i18n, etc. 
+```go
+import github.com/aerogear/charmil/pkg/factory
+```
+
+## How to use
+
+1. Create a clean cobra command for your plugin

```

Comment:
```suggestion
1. Create a cobra command for your plugin

```
we do not like dirty cobras? racial discrimination

Reply:
done